src/components/Table/TableCell/TableCell.scss

```diff
@@ -0,0 +1,30 @@
+.moonstone-table_cell {
```

Comment:
I think it's better to use the same name than the component but in camelCase: `moonstone-tableCell`.

Reply:
If we're using a CSSBEM-like naming convention, should it be like `moonstone-block-element_modifier`? Where table is the block in this case and cell is the element?

Reply:
If we follow the BEM documentation you right, but I'd like to have the component name associate with the class name. I think it easier to debug/maintain.